src/commands/job-collector.yml

```diff
@@ -0,0 +1,72 @@
+description: >
+    "Collect data from every job and report to SumoLogic"
+parameters:
+    environment:
+        default: ""
+        description: Enter the environment name of the event.
+        type: string
+    team:
+        default: ""
+        description: Enter the team carrying out the event.
+        type: string
+    service:
+        default: ""
+        description: Enter the service carrying out the event.
+        type: string
+    job-collector:
+        default: JOB_HTTP_SOURCE
+        description: Enter the HTTP Source endpoint generated by the Sumologic setup wizard for Jobs.
+        type: env_var_name
+    circle-token:
+        default: CIRCLE_TOKEN
+        description: 'Enter your CircleCI Personal Access Token for interacting with the API. You may generate one here: https://circleci.com/account/api'
+        type: env_var_name
+    
+steps:
+    - jq/install
+    - run:
+        name: Job Collector
+        command: |
+            VCS_SHORT=$(echo $CIRCLE_BUILD_URL | cut -d"/" -f4)
+            case $VCS_SHORT in
+            gh)
+                VCS=github
+                ;;
+            bb)
+                VCS=bitbucket
+                ;;
+            *)
```

Comment:
Does this mean the job collector will error out if customers are using any VCS other than github or bitbucket?

Reply:
It might be because CircleCI API below uses the VCS, and they support BB and GH.